convert.go

```diff
@@ -249,16 +253,28 @@ func convert(val string, retval reflect.Value, options multiTag) error {
 	case reflect.Slice:
 		elemtp := tp.Elem()
 
-		elemvalptr := reflect.New(elemtp)
-		elemval := reflect.Indirect(elemvalptr)
-
-		if err := convert(val, elemval, options); err != nil {
-			return err
+		parts := []string{val}
+		delim := options.Get("value-delim")
```

Comment:
We need to think about the case of users wanting to specify also a value that contains the delim (i.e. escaping the delim).